--- a/src/templates/padBootstrap.js
+++ b/src/templates/padBootstrap.js
@@ -31,7 +31,7 @@
   window.plugins.baseURL = basePath;
   await window.plugins.update(new Map([
     <% for (const module of pluginModules) { %>
-    [<%- JSON.stringify(module) %>, require("../../src/plugin_packages/"+<%- JSON.stringify(module) %>)],
+    [<%- JSON.stringify(module) %>, require("./src/plugin_packages/"+<%- JSON.stringify(module) %>)],
     <% } %>
 ]));
   // Mechanism for tests to register hook functions (install fake plugins).
